StaticHelpers/StringHelpers.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection.Metadata;
+
+namespace PhoneBook.StaticHelpers
+{
+    public static class StringHelpers
+    {
+
+        /*"N"---------- number
+          "C"---------- char
+          "(''||'')"--- Or              In dev
+         */
+
+        public static bool Validator(this string str)
```

Comment:
Make this file clean. Remove comments, change namings to have more meaningful names